libs/agno/agno/db/sqlite/sqlite.py

```diff
@@ -48,6 +48,8 @@ def __init__(
         metrics_table: Optional[str] = None,
         eval_table: Optional[str] = None,
         knowledge_table: Optional[str] = None,
+        trace_table: Optional[str] = None,
```

Comment:
`traces_table`, `spans_table`. We made a mistake with the others, so best to be correct going forward